src/features/selectedPostSlice.ts

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable no-param-reassign */
+import { PayloadAction, createSlice } from '@reduxjs/toolkit';
+import { Post } from '../types/Post';
+
+export interface AuthorState {
```

Comment:
The name of the interface 'AuthorState' does not match with the feature it represents. It should be named something like 'SelectedPostState' to accurately represent the state slice.